### DISCUSSION TITLE
Integration with Sharepoint?

### DISCUSSION THREAD

**Discussion by MalD73** [Lumieducation/Lumi#2140] [Q&A]
Has anyone successfully integrated an H5P export from Lumi into a Microsoft Sharepoint page? I've tried to embed the standard .html export, and it clearly sees the file but doesn't display the content. I've also looked into possibly taking the raw code from the html file and saving it as an .aspx file, but haven't quite figured out how to make that work (if it's even possible). Anyone have any thoughts or suggestions?

**Comment by sr258** [comment]
I'm not entirely certain but I think that embeds in a Sharepoint page don't allow arbitrary JavaScript execution.
